### DISCUSSION TITLE
Not able to deploy the learner next.js app to vercel

### DISCUSSION THREAD

**Discussion by poulami95** [vercel/next.js#30678] [Help]
Hi All,

This would sound a bit odd but i was just trying to deploy the nextjs-blog cards, build with the help of [https://nextjs.org/learn/basics/create-nextjs-app](https://nextjs.org/learn/basics/create-nextjs-app) to vercel from my git repo as a part of learning. However in Vercel when i am trying to deploy, its always coming up "Deployment failed with error". Also i am not able to view why it is failing. Can anyone please help me with this. My repo with the updated code is available here: 
[https://github.com/poulami95/learning-nextjs](https://github.com/poulami95/learning-nextjs)

**Comment by samcx** [comment]
Hi there,

Can you let us know what is your email under your Vercel account so we can take a look?
